textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team1lucenehotelexample/Indexer.java

```diff
@@ -0,0 +1,71 @@
+package edu.uci.ics.textdb.sandbox.team1lucenehotelexample;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.StringField;
+import org.apache.lucene.document.TextField;
+import org.apache.lucene.index.IndexWriter;
+import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.store.FSDirectory;
+
+/**
+ * Index all text files under a directory.
+ * <p>
+ * 
+ */
+public class Indexer {
+
```

Comment:
@JavierJia We are using space=4 for indentations. We added these files freshly from the old branch. That's why git shows them as new lines.